js/searchbar/calculatorPlugin.js

```diff
@@ -0,0 +1,50 @@
+/*
+  this plugin will provide the ability to perform simple calculations
+*/
+
+const { clipboard } = require('electron')
+const searchbarPlugins = require('searchbar/searchbarPlugins.js')
+const Parser = require('expr-eval').Parser
+
+const math = new Parser()
+
+function doMath (text, input, event) {
+  searchbarPlugins.reset('calculatorPlugin')
+  var result
+
+  try {
+    result = math.evaluate(text)
+  } catch (e) { return }
+
+  searchbarPlugins.addResult('calculatorPlugin', {
+    icon: 'carbon:calculator',
+    title: `${result.toString()}`
```

Comment:
Maybe `descriptionBlock: 'Click to Copy'` here? Most people won't hover over the search result, so that would make it clearer that the option exists. And then when you click on it, the text could change to "Copied" to provide feedback.

Reply:
That sounds like a really good idea, I will implement it.